### DISCUSSION TITLE
Possible exploit?

### DISCUSSION THREAD

**Discussion by Aka-but-nice** [3kh0/ext-remover#333] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
![Screenshot 2022-12-14 5 39 26 PM](https://user-images.githubusercontent.com/119691881/207730696-bdeaf835-d802-4e53-a8d8-10bb53503b28.png)
There has to be a possible exploit here, am I right, or am I right?

**Comment by 3kh0** [comment]
Possible

**Reply by viperbot2000** [reply]
PLZ FIGURE IT OUT!!!

**Reply by Aka-but-nice** [reply]
it takes time

**Comment by Avad3** [comment]
If there was it would've already been figured out. The description of the method makes it so there isn't any way for it to be exploited.

**Reply by Aka-but-nice** [reply]
true

**Comment by Aka-but-nice** [comment]
Refer to #347 for an update!
